### PR TITLE
add paracompact properties and arens space props

### PR DESCRIPTION
**Pull request by StevenClontz**

### CODE REVIEW

**Comment by prabau** [comment]
Nice addition for the paracompact related properties.  You kind of beat me to it for the Arens space, as I was in the middle of adding various properties as well.  Maybe I can just comment on what you have and then add more afterwards.

File: spaces/S000156/properties/P000050.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+space: S000156
+property: P000050
+value: true
+refs:
+- mathse: 4673615
+  name: What separation properites are satisfied by the Arens space?
+---
+
+Basic open sets in a convergent sequence are open; this passes to {S000156}.
```
Comment (prabau) [inline spaces/S000156/properties/P000050.md:10]: You meant "are closed" I suppose.

Also, can't we just use "{S156}" for simplicity?  Easier to read when we edit things.

File: properties/P000146.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+uid: P000146
+slug: ultraparacompact
+name: Ultraparacompact
+refs:
+  - mathse: 4673639
+    name: Answer to What separation properites are satisfied by the Arens space?
```
Comment (prabau) [inline properties/P000146.md:7]: For ultraparacompact I think a more focused ref would be the arxiv paper you mentioned, which has a doi:
https://doi.org/10.48550/arXiv.1306.6086

**Comment by prabau** [comment]
The definition of ultraparacompact in J. Van Name's paper includes Hausdorff.  As we discussed before, it allows more generality if we don't include that in the pi-base definition.  But we should make a note of this.

Do you know where the term "ultraparacompact" originated?  A google search gives some papers using it from the 1990's and 1980's and even earlier maybe.

**Comment by StevenClontz** [comment]
I don't know much about ultraparacompact; I was able to see quickly that the Arens space must have this property (whatever it was named), then found the preprint that showed the elegant sufficient conditions of Lindelof + 0-dim.

**Comment by prabau** [comment]
I made a few changes.  Please take a look.

For T343 in particular, Engelking mentions on p. 317 that a star-finite family of sets is not necessarily locally finite, but it will be the case if all the sets are open.  So I added that the sets were open.

**Comment by prabau** [comment]
Also fixed typo (properites --> properties) in the title of https://math.stackexchange.com/questions/4673615 that we are referring to.  But that should be fixed in mathse as well so things match up.

**Comment by StevenClontz** [comment]
Looks fine to me. I'll merge after getting an approving review. I requested Chris but it's okay if Patrick make the approval instead since 11f4cd7945dbed9c191c32a813de8ec99a3f485f and 313cabde277c64d70598be194ccc024e15a9f123 are pretty light amendments.

**Comment by prabau** [comment]
I'll approve, but I need to discuss one more thing.  Give me a few minutes.

**Comment by prabau** [comment]
I want to add something, but can you remind me how to search for an MR article reference number without logging in to it?

Want to get the MR ref for https://eudml.org/doc/161976.

**Comment by prabau** [comment]
Some older papers seem to use a slightly different definition for ultraparacompact in terms of a locally finite refinement instead of a partitioning refinement.  The two notions are equivalent, as show in Ellis (Math. Ann. 1970), which seems to be the first to use the terminology "ultraparacompact", at least in English.  (The article mentions Ponomarev lookes at these spaces in 1962 in Russian.)  So I added mention of both defs.

If you are ok with that, I'll approve.

**Comment by StevenClontz** [comment]
Sounds good to me.

**Review by prabau — APPROVED** [review]

**Pull request merged**